lib/serverspec/type/hdfs_config.rb

```diff
@@ -0,0 +1,22 @@
+begin
+  require 'nokogiri'
+rescue LoadError
+  fail "nokogiri is not available. Try installing it."
+end
```

Comment:
Hi

Thank you very much for the feedback. I will update the code.

Currently, the XML file name is fixed. Is there any efficient way to make this applicable to any XML file?

Thank you.